src/Microsoft.PowerShell.Commands.Management/commands/management/Computer.cs

```diff
@@ -2133,12 +2133,30 @@ internal static bool SkipSystemRestoreOperationForARMPlatform(PSCmdlet cmdlet)
 
                     if (!InternalTestHooks.TestStopComputer)
                     {
-                        CimMethodResult result = cimSession.InvokeMethod(
-                            ComputerWMIHelper.CimOperatingSystemNamespace,
-                            ComputerWMIHelper.WMI_Class_OperatingSystem,
-                            ComputerWMIHelper.CimOperatingSystemShutdownMethod,
-                            methodParameters,
-                            operationOptions);
+                        CimMethodResult result = null;
+
+                        if (isLocalhost)
+                        {
+                            // Win32_ComputerSystem is a singleton hence FirstOrDefault() return the only instance returned by EnumerateInstances.
+                            var computerSystem = cimSession.EnumerateInstances(ComputerWMIHelper.CimOperatingSystemNamespace, ComputerWMIHelper.WMI_Class_OperatingSystem).FirstOrDefault();
+
+                            // Win32_ComputerSystem is a singleton hence FirstOrDefault() return the only instance returned by EnumerateInstances.
+                            result = cimSession.InvokeMethod(
+                                ComputerWMIHelper.CimOperatingSystemNamespace,
+                                computerSystem.FirstOrDefault(),
```

Comment:
The `FirstOrDefault()` call here is redundant

Reply:
Removed duplicate and the duplicate comment as well.